database_schemas/cassandra/benchflow.cql

```diff
@@ -182,6 +186,109 @@ CREATE TABLE host_properties (
   PRIMARY KEY(host_id)
 );
 
+-- Faban
+CREATE TABLE faban_run_info (
```

Comment:
Where do you save the `status` field https://github.com/benchflow/data-transformers/issues/52